Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
 members = [
 	"bee-common",
+	"bee-signing/bee-signing-derive",
```

Comment:
Is there a special reason why we tie this to bee-signing? I understand that bee-signing is the most obvious "customer" of those proc macros, but to me it seems those could have a broader usage, like if I want to create a custom showcase wallet with a type `Balance(i64)` where I want to derive `SecretDisplay` for it. Just a though 🙂

Reply:
This is what I've seen others do, and I don't think we'll have usecases outside of signing but sure, I can make it common-derive.